### DISCUSSION TITLE
@DefaultMessage decorator not in @tsed/schema

### DISCUSSION THREAD

**Discussion by Hariram-atoa** [TypedProject/ts-express-decorators#2610] [General]
Have tried to access @DefaultMessage decorator to give default error message. But i can't able to access it from @tsed/schema why ??

[TsED Documentation](https://api-docs.tsed.io/api/specs/schema/types/decorators/common/DefaultMsg.html)

official documentation is saying that it is present...

**Comment by Romakita** [comment]
Hello @Hariram-atoa 

The correct import seems to be DefaultMsg ;) The example section seems to be wrong. Sorry !

**Comment by Hariram-atoa** [comment]
Thanks for the reply @Romakita 

I don't think DefaultMsg is also present... I have checked the packages in the node modules.

**Reply by Romakita** [reply]
export * from "./decorators/common/defaultMsg";
It's exported

**Reply by Romakita** [reply]
```
export function DefaultMsg(message: string): ClassDecorator {
  return useDecorators(ErrorMsg({_: message}));
}
```

**Reply by Hariram-atoa** [reply]
@Romakita Thanks ...

let me try on my side with the latest version :)
